src/libraries/AndGPT.Core/src/Services/EventAggregator.cs

```diff
@@ -0,0 +1,40 @@
+﻿#nullable disable
+
+using System.Diagnostics.CodeAnalysis;
+using AndGPT.Core.Contracts.Services;
+
+namespace AndGPT.Core.Services;
+
+public class EventAggregator : IEventAggregator
```

Comment:
We should either find an event aggregator like Mediatr or other more robust event bus.   This is just to get things flowing.